Pathfinder.elm

```diff
@@ -47,8 +50,12 @@ obstacleTiles obstacles =
   |> List.concat
 
 
-find : (Int, Int) -> List (Int, Int) -> (Int, Int) -> (Int, Int) -> List (Int, Int)
-find = astar
+createMap : List (Int, Int) -> (Int, Int) -> Map
+createMap = Astar.createGrid
+
+
+find : (Int, Int) -> (Int, Int) -> Map -> List (Int, Int)
+find = Astar.findPath
```

Comment:
What if we delete this file entirely and use `AStar.elm`?  All we need is `obstacleTiles` and it may be moved to where it is called.